packages/fxa-react/components/AppErrorDialog/index.stories.tsx

```diff
@@ -2,6 +2,6 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import AppErrorDialog from './index';
 
-storiesOf('components/AppErrorDialog', module).add('basic', () => (
+storiesOf('Components|AppErrorDialog', module).add('basic', () => (
```

Comment:
We probably also want to let @lmorchard know about changing any `fxa-react` Storybook references to `Components|` instead of `components/`, though his hook PR will likely make it first, we can pull in those changes and update here

Reply:
Good point. @lmorchard it's probably worth checking out how we're setting up Storybook categories now; it's pretty much just prefixing the title with `Category|`. In your case, instead of `hooks/useAwait`, it would likely be `Hooks|useAwait`

Reply:
I *did* just merge that useAwait PR, but I updated it to use `hooks|useAwait`. Of course now that I come back here, I see it probably should have been `Hooks|` with a capital `H`, though the styling does an A L L C A P S anyway? 🤷

Reply:
Yeah, not too worried about the capital. Thanks!